include/ck/utility/static_buffer.hpp

```diff
@@ -133,8 +132,7 @@ struct StaticBufferTupleOfVector
     // Set X
     // i is offset of S, not X. i should be aligned to X
     template <typename X,
-              index_t I,
-              typename enable_if<has_same_scalar_type<S, X>::value, bool>::type = false>
```

Comment:
If you need this to work with packed data type please add specialization.

Reply:
Done